pgjdbc/src/main/java/org/postgresql/jdbc/PgResultSet.java

```diff
@@ -1854,23 +1858,57 @@ private void setRowBufferColumn(Tuple rowBuffer,
         // or we won't be able to re-parse it.
         //
         case Types.DATE:
```

Comment:
Should be additionally have a `case` for `Types.TIMESTAMP_WITH_TIMEZONE` to be in sync with `org.postgresql.jdbc.PgResultSet.getObject(int, Class<T>)` and `org.postgresql.jdbc.PgPreparedStatement.setNull(int, int)` and `org.postgresql.jdbc.PgPreparedStatement.setObject(int, Object, int, int)`?